src/main/java/com/google/sps/api/tasks/TasksProvider.java

```diff
@@ -14,9 +14,36 @@
 
 package com.google.sps.api.tasks;
 
+import com.google.api.services.tasks.model.Task;
 import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
 
 public class TasksProvider {
-  public static final ImmutableList<String> TASKS_SAMPLE = ImmutableList.of(
+  public static final ImmutableList<String> TASKS_SAMPLE_TITLES = ImmutableList.of(
       "Review code", "Write the design doc", "Talk to PM", "Investigate report", "Prepare slides");
+
+
+  public static final List<Task> TASK_MODEL_LIST = new ArrayList<Task>();
+
+  public TasksProvider() {
+    int index = 0;
+    for (String title: TASKS_SAMPLE_TITLES) {
+      Task task = new Task();
+      task.setId(Integer.toString(index));
+      task.setTitle(TASKS_SAMPLE_TITLES.get(index));
+      index++;
+      TASK_MODEL_LIST.add(task);
+    }
+  }
+
+  public List<String> getSampleAsString() {
+    return TASK_MODEL_LIST.stream().map(task -> task.getTitle()).collect(Collectors.toList());
```

Comment:
This is probably redundant, since the title are already inside the `TASKS_SAMPLE_TITLES` list

Reply:
I think it could help when we get rid of the hardcoded tasks.